lib/smart_proxy_pulp_plugin/pulp_node_plugin.rb

```diff
@@ -1,7 +1,8 @@
 module PulpMasterProxy
   class Plugin < ::Proxy::Plugin
     plugin "pulpnode", ::PulpProxy::VERSION
-    default_settings :pulp_url => 'https://localhost/pulp'
+    default_settings :pulp_url => 'https://localhost/pulp',
+                     :pulp_dir => '/var/lib/pulp'
```

Comment:
@jlsherrill is there anything else we care on the filesystem in this regard? thanks